src/bindist/autoemu.bat

```diff
@@ -1,8 +1,10 @@
 @echo off
 rem generic autoexec.bat for DOSEMU + any DOS
 rem must be edited in most circumstances
-path c:\dos;c:\windows\command;e:\dosemu
-lredir d: linux\fs\${DOSEMU_LIB_DIR}/freedos ro
+set PATH=c:\windows\command;e:\dosemu

```

Comment:
I just looked at system -s, but that just seems to be for passing around environment variables. I'm not sure how you image I'd use it for this purpose?

My intention with autoemu.bat was to make it flexible so that it would automatically work for the most common situations without any editing. Using EXIST seems the most trivial way to get a proper path in all situations.

Btw. I just pushed a new version which does everything with lowercase filenames.